cdk/lib/report.ts

```diff
@@ -17,5 +35,79 @@ export class Report extends GuStack {
 		new CfnInclude(this, 'YamlTemplate', {
 			templateFile: yamlTemplateFilePath,
 		});
+		const app = 'report';
+		const playApp = new GuPlayApp(this, {
+			access: {
+				scope: AccessScope.PUBLIC,
+			},
+			app,
+			certificateProps: {
+				domainName: props.domainName,
+			},
+			instanceMetricGranularity: props.instanceMetricGranularity,
```

Comment:
We improve observability here (compared to the legacy stack) by using 1-minute granularity for `PROD`:

https://github.com/guardian/mobile-n10n/blob/99391a17ff5de34b89a78194eadba5a49d189f61/cdk/bin/cdk.ts#L90

This is recommended for all `PROD` services by DevX.